src/pytorch_lightning/plugins/environments/cluster_environment.py

```diff
@@ -63,7 +63,7 @@ def set_global_rank(self, rank: int) -> None:
 
     @abstractmethod
     def local_rank(self) -> int:
-        """The rank (index) of the currently running process inside of the current node."""
+        """The rank (index) of the currently running process of the current node."""
```

Comment:
"inside" was there to express that the local_rank is counting the ranks local to the current node.